src/google.rs

```diff
@@ -229,4 +229,95 @@ pub mod protobuf {
             })
         }
     }
+
+    mod sealed {
+        #[allow(unused_imports)]
+        use super::*;
+        use alloc::string::String;
+        use alloc::vec::Vec;
+
+        #[allow(dead_code)]
```

Comment:
Can we remove the `allow` lint attributes? 
```suggestion
    #[cfg(any(feature = "borsh", feature = "parity-scale-codec"))]
    mod sealed {
        use super::Any;

        use alloc::string::String;
        use alloc::vec::Vec;
```